gulpfile.js

```diff
@@ -218,6 +218,15 @@ gulp.task('copyLocalEnvConfig', function () {
     .pipe(gulp.dest('config/env'));
 });
 
+// Make sure upload directory exists
+gulp.task('makeUploadsDir', function () {
+  return fs.mkdir('modules/users/client/img/profile/uploads', function (res) {
```

Comment:
It's a common node convention for callbacks to return an error, so err makes more sense.